### PR TITLE
Fixes body purist headrevs from getting mad at their implant by making them unaware of it

### LINKED ISSUE #74180: Body purist quirk hates being a headrev
Reporting client version: 514.1589

## Round ID:
[202309](https://scrubby.melonmesa.com/round/202309)

## Reproduction:
After becoming a provocateur my mood quickly tanked as the antag mood buff was no match for body purist getting pissy. After revs ended suddenly the quirk was no longer acting up and everything was fine.

The quirk should probably exclude whatever being a headrev gives you.

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Body purists no longer mind implants with a special trait for "hidden implants". This special trait is replacing the "syndicate implant" variable and was just used to hide certain implants from medical scanners.

## Why It's Good For The Game

fixes #74180

## Changelog
:cl:
fix: Body purist headrevs having a bad time with their implant
/:cl:

### CODE REVIEW

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Comment by tralezab** [comment]
yeah uuuh theres some bad issues i have fixed i'll push up commits in a bit

**Comment by tralezab** [comment]
reviving...

**Comment by tralezab** [comment]
ready!

File: code/modules/surgery/organs/augments_eyes.dm [hunk 1]
```diff
@@ -53,4 +53,4 @@
 /obj/item/organ/internal/cyberimp/eyes/hud/security/syndicate
 	name = "Contraband Security HUD Implant"
 	desc = "A Cybersun Industries brand Security HUD Implant. These illicit cybernetic eye implants will display a security HUD over everything you see."
-	syndicate_implant = TRUE
+	organ_flags = ORGAN_HIDDEN
```
[the remark this answers is not in the document]
Comment (tralezab) [reply code/modules/surgery/organs/augments_eyes.dm:56]: 😭

**Pull request merged**